src/app/colours.scss

```diff
@@ -38,19 +46,30 @@
 
   // Icons
   --od-colour-icon-neutral: var(--nw-colour-icon-neutral, #fff);
+  --od-colour-icon-inverse: var(--nw--colour-icon-inverse, #ffffff);
 }
 
 [data-scheme='dark'] {
   // Surfaces
+  --od-colour-surface-neutral: var(--bg, var(--nw-colour-surface-neutral, rgb(15, 15, 15)));
   --od-colour-surface-utility: #30323699;
+  --od-colour-surface-utility-opaque: var(--nw-colour-surface-utility, #303236);
   --od-colour-surface-utility-bolder: #18191bbf;
+  --od-colour-border-opacity-low: var(--od-colour-surface-utility-opaque, #303236);
```

Comment:
Looks like this should be: `--nw-colour-border-opacity-low` ? @drzax

Reply:
I should have put a comment in — after much troubleshooting, that follows the styles for share buttons in FN. It's an inconsistency which should be fixed, but I didn't have the stamina last night.

Reply:
Ah, but it doesn't follow the styles of the WISYWIG embed, which _should_ use that same `nw` var in both light and dark mode!

Reply:
ie. https://github.com/abcnews/odyssey/blob/1bd7616965b670948adc2a6ad8d19ec42e7d3431/src/app/components/utilities/richtext.scss#L417

Reply:
I'll take another look.